src/Microsoft.DotNet.XHarness.iOS/ErrorKnowledgeBase.cs

```diff
@@ -13,50 +13,52 @@ namespace Microsoft.DotNet.XHarness.iOS
 {
     public class ErrorKnowledgeBase : IErrorKnowledgeBase
     {
-        const string IncorrectArchPrefix = "IncorrectArchitecture";
+        static readonly Dictionary<string, string> testErrorMaps = new Dictionary<string, string> {
```

Comment:
nit: formatting in this file - missing braces and braces on new lines mostly, some missing accessibility modifiers..